.github/workflows/main.yml

```diff
@@ -10,8 +10,12 @@ jobs:
   tests:
     if: github.event.action != 'closed' || github.event.pull_request.merged
 
+    name: Test Python ${{ matrix.python_version }}
     runs-on: ubuntu-latest
-
+    strategy:
+      max-parallel: 4
```

Comment:
Why the limit?

Reply:
Really no need, I can definitely remove it.

Perhaps I was drinking the "Explicit is better than implicit" sauce here (we need only exactly 4 here).